pkg/mimir/mimir.go

```diff
@@ -750,6 +752,10 @@ func New(cfg Config, reg prometheus.Registerer) (*Mimir, error) {
 
 	mimir.setupObjstoreTracing()
 	otel.SetTracerProvider(NewOpenTelemetryProviderBridge(opentracing.GlobalTracer()))
+	//middleware.InitGRPCMiddleware(&mimir.Cfg.Server)
+
+	mimir.Cfg.Server.Router = mux.NewRouter()
```

Comment:
Is this router used for direct HTTP requests as well as handling `httpgrpc` requests? If so, do we need to do something in the middleware to only add the tags to the parent span and create the child span if we're handling a `httpgrpc` request?

Reply:
Will look closer at this